src/devices/README.md

```diff
@@ -211,6 +211,7 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 * [Adafruit Seesaw - extension board (ADC, PWM, GPIO expander)](Seesaw/README.md)
 * [MotorHat](MotorHat/README.md)
 * [Pca9685 - I2C PWM Driver](Pca9685/README.md)
+* [uFire_ISE - I2C Driver](uFire_ISE/README.md)
```

Comment:
Same, please don't change this file, it is changed automatically. Please add a txt file at root of the directory, you'll find examples in all bindings.